debezium-core/pom.xml

```diff
@@ -47,6 +47,11 @@
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-jsr310</artifactId>
         </dependency>
+        <dependency>
```

Comment:
What is its license? And do you happen to know whether this has been productized before?

Reply:
Goovy is ASL - but I'd prefer not to pack it into the package but handle it as bring your own JAR to keep the connector lightwight.

Reply:
Ok, I'd prefer that, too. This also ties into my comment on JSR 223 below.